ci/build-all-steps.yml

```diff
@@ -28,6 +28,7 @@ steps:
 # Slice the tasks (sets TASK_PATTERN)
 - script: node ./ci/set-task-slice.js
   displayName: Set task slice
+  # condition: ne(variables.DISTRIBUTEDTASK_USE_PERTASK_NUGET, 'true')
```

Comment:
another option would be to produce both layouts for a sprint on the tasks repo, and we can have an env var on the vso side. producing the extra per task layout should be cheap (eixisting functions use symlink/hardlink instead of copying files)

Reply:
Good point, that definitely makes sense. Was worried it would slow total build time down too much. Will get rid of this. I have the env var downstream.

Reply:
Removed.